sympy/integrals/transforms.py

```diff
@@ -1014,9 +1034,42 @@ def expand_dirac_delta(expr):
     """
     return _lin_eq2dict(expr, expr.atoms(DiracDelta))
 
+def laplace_transform_replace_ic(expr, f, F, t, s, ic):
```

Comment:
Should this function be considered public? In other words do we expect users to use this function? If so it should have better docs following the style guide:
https://docs.sympy.org/latest/documentation-style-guide.html

Reply:
Yes, it should, so I will write an appropriate doc with example.